.github/workflows/validate_internal_docs_links.yml

```
@@ -0,0 +1,23 @@
+name: Validate internal documentation links
+
+"on": [pull_request]
+
+jobs:
+    tests:
+        name: mkdocs serve strict
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4.0.0
```

How about we keep it at major versions so fixes appear automagically? 😊 Which also reminds that it's time to update a few actions, like checkout - thats like two weeks out.. 😁 
```suggestion
            - uses: actions/checkout@v4
```

---

Agreed!